lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -8,8 +8,7 @@
 /* global window, document, getNodeDetails */
 
 const Gatherer = require('./gatherer.js');
-const fs = require('fs');
-const axeLibSource = fs.readFileSync(require.resolve('axe-core/axe.min.js'), 'utf8');
+const axeLibSource = require('../../lib/axe.js').source;
```

Comment:
I was scared of the changes in this file until I did "Hide whitespace changes". Nice cleanup! Much more readable.